android/app/src/main/java/com/chorestory/adapter/ParentHomeAdapter.java

```diff
@@ -13,12 +13,20 @@
     private List<Fragment> fragmentList;
```

Comment:
This file is literally a copy-paste of the `ChildHomeAdapter`, get rid of one of them